Gems/Atom/Tools/AtomToolsFramework/Code/Source/EntityPreviewViewport/EntityPreviewViewportSettingsSystem.cpp

```diff
@@ -589,6 +589,7 @@ namespace AtomToolsFramework
         AZ::Job* job = AZ::CreateJobFunction(
             [toolId = m_toolId]()
             {
+                AZ_TracePrintf("EntityPreviewViewportSettingsSystem", "Enumerating presets started.");
```

Comment:
Are these printfs debugging code?

Reply:
Yes and no. I'm mostly interested in the time stamps for when these events begin and end so users experiencing extended startup times can provide the log files with the information from that specific run. Honestly, I should sprinkle more of these to capture additional data during initialization. These are the areas that I suspected might cause a delay based on the size of the project, other than connecting to the AP, which has become negligible since optimizations, and initializing other project specific gems and features. However, the number is negligible on the hardware that I have tested, going between automated testing and MPS. MPS only takes slightly longer because it's initializing the server, splash screen, and other systems that could probably check for editor, launcher, game mode.